app/routes/events/components/JoinEventFormCountdownProvider.js

```diff
@@ -30,7 +30,7 @@ const COUNTDOWN_INTERVAL = 1000;
 // Must be sorted lo->hi
 const TICK_ACTIONS: Array<[number, Action]> = [
   [0, 'REGISTRATION_AVAILABLE'],
-  [60 * 1000, '1_MINUTE_LEFT'],
+  [180 * 1000, '3_MINUTE_LEFT'],
```

Comment:
3 * 60 * 1000 for maximum readability? 😛

Reply:
Fair 8-)